examples/cs/core/inheritance/program.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Examples
+{
+    using System;
+    using System.Diagnostics;
+    using Bond;
+    using Bond.Protocols;
+    using Bond.IO.Unsafe;
+
+    using examples.inheritance;
+
+    static class Program
+    {
+        static void Main()
+        {
+            var derivedObj = new Derived {str = "derived"};
+            ((Base) derivedObj).str = "derived base";
+            var derivedPayload = new CompactBinaryReader<InputBuffer>(
+                new InputBuffer(Serialize(derivedObj)));
+
+            {
```

Comment:
Forgot to clean these up when porting the C++ example. It has two example sections, while C# and Java only have one.